exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/CodedOutputStream.java

```diff
@@ -58,7 +58,7 @@
 //
 // Unneeded lines of code are deleted as is, without any modifications otherwise.
 @SuppressWarnings({"UngroupedOverloads", "InlineMeSuggester"})
-abstract class CodedOutputStream {
+public abstract class CodedOutputStream {
```

Comment:
we should probably add our standard boilerplate for internal classes in javadoc here.